cli/src/pcluster/aws/aws_resources.py

```diff
@@ -164,28 +164,85 @@ def __init__(self, instance_type_data):
 
     def gpu_count(self):
         """Return the number of GPUs for the instance."""
-        # FixMe: this method is not used in the pcluster3 CLI
         gpu_info = self.instance_type_data.get("GpuInfo", None)
 
         gpu_count = 0
         if gpu_info:
-            for gpus in gpu_info.get("Gpus", []):
-                gpu_manufacturer = gpus.get("Manufacturer", "")
-                if gpu_manufacturer.upper() == "NVIDIA":
-                    gpu_count += gpus.get("Count", 0)
-                else:
-                    LOGGER.warning(
-                        "ParallelCluster currently does not offer native support for '%s' GPUs. "
-                        "Please make sure to use a custom AMI with the appropriate drivers in order to leverage "
-                        "GPUs functionalities",
-                        gpu_manufacturer,
-                    )
+            for gpu in gpu_info.get("Gpus", []):
+                manufacturer = gpu.get("Manufacturer", "")
+                if manufacturer.upper() == "NVIDIA":
+                    gpu_count += gpu.get("Count", 0)
 
         return gpu_count
 
+    def gpu_manufacturer(self) -> str:
+        """Return the list of GPU manufacturers supported by this instance type."""
+        gpu_info = self.instance_type_data.get("GpuInfo", None)
+
+        gpu_manufacturers = list({gpu.get("Manufacturer", "") for gpu in gpu_info.get("Gpus", [])}) if gpu_info else []
+
+        # Only one GPU manufacturer is associated with each Instance Type's GPU
+        manufacturer = gpu_manufacturers[0] if gpu_manufacturers else ""
+        if manufacturer.upper() != "NVIDIA":
+            LOGGER.warning(
+                "ParallelCluster currently does not offer native support for '%s' GPUs. "
+                "Please make sure to use a custom AMI with the appropriate drivers in order to leverage "
+                "GPUs functionalities",
+                manufacturer,
+            )
+        return manufacturer
+
+    def inference_accelerator_manufacturer(self) -> str:
+        """Return the list of Inference Accelerator Manufacturers supported by this instance type."""
+        inference_accelerator_info = self.instance_type_data.get("InferenceAcceleratorInfo", None)
+
+        inference_accelerator_manufacturers = (
+            list(
+                {
+                    accelerator.get("Manufacturer", "")
+                    for accelerator in inference_accelerator_info.get("Accelerators", [])
+                }
+            )
+            if inference_accelerator_info
+            else []
+        )
+        # Only one accelerator manufacturer is associated with each Instance Type's accelerator
+
+        accelerator_manufacturer = inference_accelerator_manufacturers[0] if inference_accelerator_manufacturers else ""
+        if accelerator_manufacturer.upper() != "AWS":
```

Comment:
I would say yes, since there is an accelerator data but the manufacturer is missing

Reply:
Good point. I think logging that only 'AWS' manufacturer is supported, then show the `accelerator_info`.
This will be enough to communicate why the accelerator wasn't considered.